src/index.js

```diff
@@ -1,32 +1,65 @@
+//@flow
+
 import assert from 'assert';
+
+import FusionApp from 'fusion-core';
+import type {FusionPlugin} from 'fusion-core';
+
 import {mockContext, renderContext} from './mock-context.js';
 import simulate from './simulate';
 
-export function request(app, url, options = {}) {
+declare var __BROWSER__: boolean;
+
+const request = (app: FusionApp) => (
+  url: string,
+  options: * = {}
+): Promise<*> => {
   if (__BROWSER__) {
     throw new Error(
       '[fusion-test-utils] Request api not support from the browser. Please use `render` instead'
     );
   }
   const ctx = mockContext(url, options);
   return simulate(app, ctx);
-}
+};
 
-export function render(app, url, options = {}) {
+const render = (app: FusionApp) => (
+  url: string,
+  options: * = {}
+): Promise<*> => {
   const ctx = renderContext(url, options);
   return simulate(app, ctx);
+};
+
+export function registerAsTest(
```

Comment:
Open to thoughts here on the exact API and naming this function.  Primary motivation here is to encapsulate resolving the app to allow consumers to have a single app fixture and call `.request` or `.render` multiple times (which you cannot do today).

Reply:
`createSimulator`? `createTestDriver`?